src/openzaak/components/documenten/admin.py

```diff
@@ -251,3 +253,23 @@ def get_object_actions(self, obj):
             link_to_related_objects(Gebruiksrechten, obj.canonical),
             link_to_related_objects(ObjectInformatieObject, obj.canonical),
         )
+
+    def formfield_for_dbfield(self, db_field, request, **kwargs):
+        field = super().formfield_for_dbfield(db_field, request, **kwargs)
+        private_media_fields = self.get_private_media_fields()
+        if db_field.name in private_media_fields:
+            view_name = self._get_private_media_view_name(db_field.name)
+            obj = self.get_object(request, request.resolver_match.kwargs["object_id"])
+            field.widget = PrivateFileWidget(
+                url_name="admin:%s" % view_name,
+                attrs={"display_value": obj.bestandsnaam},
```

Comment:
@sergei-maertens there has to be a better way to change the display value for the download link of the file in the admin right? I want it to show the `bestandsnaam` instead of the path where it is stored in private_media